test/fixtures/output/byok/describe-gcp-json.golden

```diff
@@ -2,7 +2,7 @@
   "id": "cck-004",
   "key": "projects/exampleproject/locations/us-central1/keyRings/testkeyring/cryptoKeys/testbyokkey/cryptoKeyVersions/3",
   "roles": [""],
```

Comment:
Unrelated to your PR, but this is kind of weird. I'm not sure we'd ever expect the backend to return a list containing an empty string... Maybe we can fix that in a separate PR.